SETUP.md

```diff
@@ -66,6 +66,18 @@ To perform backup, BBQ needs rights to read BigQuery data from the project which
     ```
 * (Optionally) Configure schedule time and kinds to export in [cron.yaml](./config/cron.yaml) file.
 
+### Security setup
+  Secure your application by following given steps.
+  * Restrict IAM roles for your GAE service account and Enable firewall for GAE application.
+    * GAE Editor needs to be removed manually, as it is set by default.
+    * You should follow PoLP (Principle of least privilege) during process or set up IAM's.
+      We recommend to set up following roles: **BigQuery Data Editor**, **BigQuery Job User**, **Cloud Datastore Import Export Admin**, **Cloud Datastore User**, **Storage Admin**, **BigQuery Data**.
```

Comment:
It's not recommended, but required. I would just say that you need to run terraform without specifying all the roles.

Reply:
👍